index.html

```diff
@@ -24,8 +27,14 @@ <h1>I'm Salma Glad to see you!</h1>
             If you like what you see and have
              a project you need coded, 
             don't hestiate to contact me.</p>
-            <h3>LET'S CONNECT</h3>
-            <img src="ic-social media icons.png" alt="social-media icons">
+            <button type="button" class="btn">LET'S CONNECT</button>
+            <ul class="social-media">
+                <a href="#"><li><i class="fa-brands fa-twitter icon"></i></li></a>
+                <a href="#"><li><i class="fa-brands fa-linkedin-in icon"></i></li></a>
+                <a href="#"><li><i class="fa-sharp fa-solid fa-m icon"></i></li></a>
+                <a href="#"><li><i class="fa-brands fa-github icon"></i></li></a>
+                <a href="#"><li><i class="fa-solid fa-hand-peace icon"></i></li></a>
+            </ul>
```

Comment:
- Great job making the changes as requested by the previous reviewer, however, let's improve it to enforce the correct semantic layout patterns below as :

*Update this*

```
<a href="#"><li><i class="fa-brands fa-twitter icon"></i></li></a>
 <a href="#"><li><i class="fa-brands fa-linkedin-in icon"></i></li></a>
 <a href="#"><li><i class="fa-sharp fa-solid fa-m icon"></i></li></a>
 <a href="#"><li><i class="fa-brands fa-github icon"></i></li></a>
 <a href="#"><li><i class="fa-solid fa-hand-peace icon"></i></li></a>
```

**TO THIS [duplicate for the remaining icons]**

```
 <li>
     <a href="" target="_blank" rel="noopener noreferrer">
        <img class="s-m-icon" src="" alt="">
     </a>
 </li>
```

- Please, let's use the correct images from the Figma as used in the design to make your design consistent following the template below:

![Screenshot from 2023-01-12 12-12-35](https://user-images.githubusercontent.com/59397159/212052336-dc1e06d7-7833-43e4-b51a-9711ce7ef94d.png)